client/src/App.js

```diff
@@ -34,181 +35,151 @@ const styles = theme => ({
-  // const getStudents = options => {
-  //   axios
-  //     .get('https://refreshr.herokuapp.com/students', options)
-  //     .then(res => {
-  //       console.log('s', res.data.students);
-  //       setStudents(res.data.students);
-  //     })
-  //     .catch(err => {
-  //       console.log(err);
-  //     });
-  // };
-
-  // //all teachers
-  // const getTeachers = options => {
-  //   axios
-  //     .get('https://refreshr.herokuapp.com/teachers', options)
-  //     .then(res => {
-  //       console.log('t', res.data.teachers);
-  //       setTeachers(res.data.teachers);
-  //     })
-  //     .catch(err => {
-  //       console.log(err);
-  //     });
-  // };
-
   /* ROUTES */
   return (
-    console.log('APP:', props.theme),
-    console.log('APP:', props.Url),
-    (
-      <>
-          <Grid
-            container
-            direction="column"
-            spacing={0}
-            justify="space-between"
-            alignItems="center"
-            className={classes.container}
-          >
-            <Grid item>
-              <Navbar theme={props.theme} {...props} />
-              <Navcrumbs {...props} />
-            </Grid>
-            <Route
-                exact path="/"
-                render={props => <LandingPage {...props} />}
-            />
-            <Grid item className={classes.routes}>
-             
-              <Route
-                path="/dashboard"
-                render={props => (
-                  <Dashboard getClasses={getClasses} allClasses={allClasses} />
-                )}
-              />
-              <Route
-                exact
-                path="/refreshrs"
-                render={props => (
-                  <RefreshrListView
-                    getRefreshrs={getRefreshrs}
-                    refreshrs={refreshrs}
-                  />
-                )}
-              />
-              <Route path="/billing" render={props => <BillingPage />} />
-              <Route
-                exact
-                path="/classes"
-                render={props => <ClassListView />}
-              />
-              <Route
-                exact
-                path="/classes/edit/:id"
-                render={props => <ClassEditView {...props} />}
-              />
-              <Route
-                exact
-                path="/classes/create"
-                render={props => <ClassCreateView />}
+    //console.log('APP:', props.theme),
+    <>
+      <Grid
+        container
+        direction="column"
+        spacing={0}
+        justify="space-between"
+        alignItems="center"
+        className={classes.container}
+      >
+        <Grid item>
+          <Navbar theme={props.theme} {...props} />
+          <Navcrumbs {...props} />
+        </Grid>
+        <Route exact path="/" render={props => <LandingPage {...props} />} />
+        <Grid item className={classes.routes}>
+          <Route
+            path="/dashboard"
+            render={props => (
```

Comment:
Looks like really just a clean up of app.js and adding to the props that are getting passed.